src/ucp/core/ucp_context.h

```diff
@@ -260,7 +260,7 @@ typedef struct ucp_context {
     /* All configurations about multithreading support */
```

Comment:
I don't understand the comment.  Is something like

/* Configuration of multi-threadiing support */

meant?

Reply:
Done.